### ISSUE TITLE
Divergence of implementation between Windows and non-Windows

### ISSUE THREAD

**Issue by leezu** [apache/incubator-mxnet#18281]
https://github.com/apache/incubator-mxnet/blob/f00b9ab5b4410a91a8f6581da696a92f85fbccf6/src/operator/numpy/np_elemwise_broadcast_op.cu#L32-L41

Shouldn't be needed anymore as 64-bit toolchain is now required on Windows? (Added due to compiler OOM errors when 32-bit toolchain was in use)

**Comment by leezu** [comment]
In GluonNLP Windows CI, tests fail on Windows with errors like

```
2020-05-13T12:06:33.5890094Z >           raise get_last_ffi_error()
2020-05-13T12:06:33.5890252Z E           mxnet.base.MXNetError: Traceback (most recent call last):
2020-05-13T12:06:33.5890363Z E             File "c:\jenkins\workspace\mxnet\mxnet\src\imperative\./imperative_utils.h", line 251
2020-05-13T12:06:33.5893446Z E           MXNetError: Check failed: outputs[i]->dtype() == out_types[i] (0 vs. 1) : 0-th output has invalid dtype. Expecting 1 got 0 in operator _npi_uniform
```

This may be due to the diverging implementation described above, or there are more issues. @zhreshold @yzhliu

**Comment by yzhliu** [comment]
@BenjaminCHEN2016 is helping.

**Comment by yzhliu** [comment]
closed by #18523
